src/contrib/persistence/Akka.Persistence.Sqlite.Tests/Batching/BatchingSqliteJournalSpec.cs

```diff
@@ -42,5 +42,8 @@ class = ""Akka.Persistence.Sqlite.Journal.BatchingSqliteJournal, Akka.Persistenc
                     }
                 }");
         }
+
+        // TODO: hack. Replace when https://github.com/akkadotnet/akka.net/issues/3811
+        protected override bool SupportsSerialization => false;
```

Comment:
Another reason for https://github.com/akkadotnet/akka.net/issues/3811 - none of the Akka.Persistence.Sql journals support `WriterGuid` at the moment due to their hand-rolled serialization layers, so the serialization specs will not pass.